session/menderclient.go

```diff
@@ -35,6 +36,18 @@ func MenderClient() Constructor {
 	return func() SessionHandler { return f }
 }
 
+func getMenderClientBinaryPath() (string, error) {
+	candidates := []string{"/usr/bin/mender-update", "/usr/bin/mender"}
```

Comment:
Q: Is it not better to check if they are in the `PATH` ? :thinking:

Reply:
Maybe you are right. Something like [this](https://github.com/mendersoftware/mender-artifact/blob/master/utils/binpath.go#L29)?

Is it okey if I push this a bit down the road? I make note as a low priority.

Reply:
Absolutely - it is not a regression in any way :)

Reply:
> Maybe you are right. Something like [this](https://github.com/mendersoftware/mender-artifact/blob/master/utils/binpath.go#L29)?

Yes :)